sdk/storage/azure-storage-files-shares/test/ut/share_client_test.cpp

```diff
@@ -523,14 +523,21 @@ namespace Azure { namespace Storage { namespace Test {
     auto shareName = m_testNameLowercase;
     auto shareClient = Files::Shares::ShareClient::CreateFromConnectionString(
         PremiumFileConnectionString(), shareName, m_options);
-    EXPECT_NO_THROW(shareClient.Create());
+    // create works
+    Files::Shares::CreateShareOptions options;
+    options.EnabledProtocols = Files::Shares::Models::ShareProtocols::Nfs;
+    options.RootSquash = Files::Shares::Models::ShareRootSquash::AllSquash;
+    EXPECT_NO_THROW(shareClient.Create(options));
```

Comment:
Don't add tests like this. We don't know if the default options would work.

Create another share.

Reply:
Also need to get share properties and compare the value of `EnabledProptocols` and `RootSquash`.

Reply:
fix it and add other two statement blocks to test nfs and smb protocols.